docs/configuration.html

```diff
@@ -70,9 +70,13 @@
 
 <h3><a id="consumerconfigs" href="#consumerconfigs">3.3 Consumer Configs</a></h3>
 
-We introduce both the old 0.8 consumer configs and the new consumer configs respectively below.
+We introduce both the new Java consumer configs and the older Scala consumer configs respectively below.
 
-<h4><a id="oldconsumerconfigs" href="#oldconsumerconfigs">3.3.1 Old Consumer Configs</a></h4>
+<h4><a id="newconsumerconfigs" href="#newconsumerconfigs">3.3.1 New Consumer Configs</a></h4>
+In 0.9.0.0 we introduced a replacement for simple and high-level consumers. Below is the configuration for the new consumer:
```

Comment:
Maybe "In 0.9.0.0 we introduced the new Java consumer as a replacement for the older Scala-based simple and high-level consumers." 

Also, would it make sense to move this to the paragraph above ("We introduce both...")?